drivers/disk/usdhc.c

```diff
@@ -12,10 +12,19 @@
 #include <sys/byteorder.h>
 #include <soc.h>
 #include <drivers/clock_control.h>
+#include <fsl_power.h>
+#if CONFIG_SDMMC_USDHC_PMIC_1_8_VOL
+#include <fsl_pca9420.h>
```

Comment:
Do we need this include?

Reply:
the fsl_pca9240 is required for the constants kPCA9240_ModeX, but you're right that the fsl_power include is a duplicate

Reply:
My concern is including a board specific include in this driver. It is possible in future we could have a different PMIC on a board. Can we add defines in a board specific header file e.g: `boards/arm/mimxrt685_evk/board.h`